.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+end_of_line = lf
```

Comment:
We use operating system native end of line markers, i.e. files are not LF on Windows. Does this confuse line endings on Windows on editors that acknowledge this file?